app/controllers/fhir/r4/api_controller.rb

```diff
@@ -426,6 +426,123 @@ def all
     status_ok(bundle) && return
   end
 
+  # Kick off async generation of bulk FHIR data for all accessible patients
+  #
+  # GET /fhir/r4/Patient/$export
+  def bulk_data_export
+    # Require all scopes for all possible resources being returned
+    return if doorkeeper_authorize!(*PATIENT_READ_SCOPES)
+    return if doorkeeper_authorize!(*OBSERVATION_READ_SCOPES)
+    return if doorkeeper_authorize!(*QUESTIONNAIRE_RESPONSE_READ_SCOPES)
+    return if doorkeeper_authorize!(*RELATED_PERSON_READ_SCOPES)
+    return if doorkeeper_authorize!(*IMMUNIZATION_READ_SCOPES)
+    return if doorkeeper_authorize!(*PROVENANCE_READ_SCOPES)
+
+    unless accept_header?
+      status_not_acceptable_with_custom_errors(["'Accept' header must have a value of 'application/fhir+json'," \
+        " or the '_format' parameter must one of 'json', 'application/json' or 'application/fhir+json'"]) && return
```

Comment:
I think you left out the word "be" `...must be one of 'json'...`

Reply:
Oops! Thanks for catching that. Fixed in https://github.com/SaraAlert/SaraAlert/pull/987/commits/c0c062f2ccf371eb58fd60424b882fed4b062731.